Tech@HubAPI/Tech@HubAPI/Models/DatabaseContext.cs

```diff
@@ -44,6 +50,25 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 entity.HasOne(e => e.User);
                 entity.HasOne(e => e.Repository);
             });
+
+            // Issue
+            modelBuilder.Entity<Issue>(entity =>
+            {
+                entity.HasKey(e => e.Id);
+            });
+
+            // Comment
+            modelBuilder.Entity<Comment>(entity =>
+            {
+                entity.HasKey(e => e.Id);
+                entity.HasOne(e => e.Body);
```

Comment:
I did some testing and it looks like this `entity.HasOne(e => e.Body);` line is causing the issues. The `HasOne` function should only be used when linking two different models together. For example line 51 is linking the `Permission` and `Repository` models. Since `Body` is just a string there is no linking to be done, this line can just be removed.